.goreleaser.yml

```diff
@@ -0,0 +1,55 @@
+before:
+  hooks:
+    - go mod tidy    
+    - go generate ./...
+builds:
+  - id: epithet-oidc-plugin
```

Comment:
Do you wanna publish the policy binary too?

Reply:
Yes, we should

Reply:
I don't consider it's a library for the lambda deployment, it can be deployed in production by your favorite method.